app/dl_formats.py

```diff
@@ -0,0 +1,85 @@
+def get_format(format: str, quality: str) -> str:
+    """
+    Returns format for download
+
+    Args:
+      format (str): format selected
+      quality (str): quality selected
+
+    Raises:
+      Exception: unknown quality, unknown format
+
+    Returns:
+      dl_format: Formatted download string
+    """
+    audio_fmt = ""
+    video_fmt = ""
+    final_fmt = ""
+
+    if format.startswith("custom: "):
+        final_fmt = format[7:]
+    elif format == "any":
+        final_fmt = "bv*+ba/b"
```

Comment:
Why the change from the previous format selections? (I mean why `bv*+ba/b`?)

Reply:
The way I've done it is so that it will choose the best format possible and then in post, will merge video/audio and convert to what ever format is needed.

I originally did this as some videos might not be available in mp4 formats (if I understand yt_dlp correctly, which I probably don't)

I can convert back to the way it was done originally if you prefer